docs/project/development.md

```diff
@@ -376,135 +239,84 @@ $ make release
-
-Bun relies on C++20 features like `std::span`, which are not available in GCC versions lower than 11. GCC 10 doesn't have all of the C++20 features implemented. As a result, running `make setup` may fail with the following error:
-
-```
-fatal error: 'span' file not found
-#include <span>
-         ^~~~~~
-```
-
-To fix the error, we need to update the GCC version to 11. To do this, we'll need to check if the latest version is available in the distribution's official repositories or use a third-party repository that provides GCC 11 packages. Here are general steps:
+If you see a 'span' file not found error on Ubuntu, it may be due to GCC's `libstdc++` library. Bun relies on C++20 features not present in GCC versions below 11. To fix this, you can upgrade GCC to version 11:
 
 ```bash
 $ sudo apt update
 $ sudo apt install gcc-11 g++-11
-# If the above command fails with `Unable to locate package gcc-11` we need
-# to add the APT repository
-$ sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test
-# Now run `apt install` again
-$ sudo apt install gcc-11 g++-11
-```
-
-Now, we need to set GCC 11 as the default compiler:
-
-```bash
 $ sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-11 100
 $ sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-11 100
 ```
 
 ### libarchive
 
-If you see an error when compiling `libarchive`, run this:
+If you encounter issues with `libarchive` during compilation, run:
 
 ```bash
 $ brew install pkg-config
 ```
 
-### missing files on `zig build obj`
+### Missing Files in 'zig build obj'
 
-If you see an error about missing files on `zig build obj`, make sure you built the headers.
+If you see missing files in 'zig build obj', ensure you've built the headers:
 
 ```bash
 $ make headers
 ```
 
-### cmakeconfig.h not found
+### cmakeconfig.h Not Found
 
-If you see an error about `cmakeconfig.h` not being found, this is because the precompiled WebKit did not install properly.
+If you encounter an error about 'cmakeconfig.h' not being found, run:
 
 ```bash
 $ bun install
 ```
 
-Check to see the command installed webkit, and you can manully look for `node_modules/bun-webkit-{platform}-{arch}`:
-
-```bash
-# this should reveal two directories. if not, something went wrong
-$ echo node_modules/bun-webkit*
-```
-
-### macOS `library not found for -lSystem`
+### macOS 'Library Not Found for -lSystem'
 
-If you see this error when compiling, run:
+If you encounter the 'library not found for -lSystem' error on macOS, run:
 
 ```bash
 $ xcode-select --install
 ```
 
-## Arch Linux / Cannot find `libatomic.a`
+## Arch Linux - Cannot Find 'libatomic.a'
 
-Bun requires `libatomic` to be statically linked. On Arch Linux, it is only given as a shared library, but as a workaround you can symlink it to get the build working locally.
+On Arch Linux, Bun requires 'libatomic' to be statically linked. You can create a symlink to make it work locally:
```

Comment:
why was the note about libatomic only being available as shared library removed.